docs/portal/audit-trail.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Audit Trail
+description: Set up and manage contacts for UID2 updates.
```

Comment:
Looks like this is the same description from email contacts

Reply:
good catch - fixed!